Taskfile.yaml

```diff
@@ -157,6 +158,10 @@ tasks:
 
       - "{{ .TOOL_DIR }}/goreleaser release --clean --skip=publish --skip=sign --snapshot --config {{ .TMP_DIR }}/goreleaser.yaml"
 
+  assert-windows-build:
+    desc: Assert that binny compiles on Windows
```

Comment:
instead of a new assertion, we are already making snapshot builds with goreleaser -- I think it makes sense to add that build configuration (we'd be blocked from releasing until we're sure about windows support, but releases in this repo are much less frequent)

Reply:
Do you have objections to merging this as is while we work on Windows support? There's no reason to be blocked on releasing on purpose - this extra task doesn't really add any time, and it's a nice way to enforce binny's current status which is, "releases of mac and linux, builds for windows but not fully supported yet."

Any objections to merging as is, and then changing goreleaser config when we're ready to officially support binny on Windows?